.github/workflows/release-chart.yaml

```diff
@@ -50,36 +53,38 @@ jobs:
 
           if [[ -z "${chart_version_match}" ]] || [[ -z "${app_version_match}" ]]
           then
-            echo "::set-output name=continue::${{ toJSON(false) }}"
+            echo "continue=${{ toJSON(false) }}" >> $GITHUB_OUTPUT
           else
-            echo "::set-output name=continue::${{ toJSON(true) }}"
+            echo "continue=${{ toJSON(true) }}" >> $GITHUB_OUTPUT
           fi
 
-      - name: Install Helm
+      - name: Set-up Helm
         if: fromJSON(steps.check_can_release.outputs.continue)
-        uses: azure/setup-helm@v1
+        uses: azure/setup-helm@f382f75448129b3be48f8121b9857be18d815a82 # v3.4
         with:
-          version: 3.*
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
why do we need the token here?

Reply:
GitHub actions which make requests to GitHub usually want the action token (this is the auto-generated workflow token) to authenticate the request so they don't get rate limited or blocked.